code/__DEFINES/is_helpers.dm

```diff
@@ -68,7 +68,7 @@ GLOBAL_LIST_INIT(turfs_without_ground, typecacheof(list(
 #define isplasmaman(A) (is_species(A, /datum/species/plasmaman))
 #define ispodperson(A) (is_species(A, /datum/species/pod))
 #define isflyperson(A) (is_species(A, /datum/species/fly))
-#define isjellyperson(A) (is_species(A, /datum/species/jelly))
+#define isjellyperson(A) (is_species(A, /datum/species/jelly) || /datum/species/feral/slime)
```

Comment:
you need to encapsulate the `/datum/species/feral/slime` with another `is_species()` check, like seen on line 84 with `isrobotic`

Reply:
Strangely enough it worked on the test server I ran, but thanks for catching it